thread-sanitizer-ignorelist.txt

```diff
@@ -0,0 +1,13 @@
+# TSan detects a race on pistache shutdown
+race:*Pistache*
+race:close
+race:socket
+# Ignore ZeroMQ races
+race:zmq::*
+# Config only changes in tests, and in places where being slightly racy doesn't matter
+race:faabric::util::SystemConfig::*
+# Catch2 allocates in its signal handler, this prevents showing the wrong crash report
+signal:*
+
+# TODO: Remove: There's something weird going on with MPI code I don't understand
```

Comment:
So for me to see the races tsan is complaining about I'd just have to re-run commenting out the next line right?

Reply:
Yes, it would terminate on the first detected race. For me that was in the AllGather implementation